package/haproxy/Dockerfile

```diff
@@ -27,9 +28,15 @@ ENV TINI_VERSION v0.10.0
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
 RUN chmod +x /tini
 
+ENV SSL_SCRIPT_COMMIT 98660ada3d800f653fc1f105771b5173f9d1a019
+RUN wget -O /usr/bin/update-rancher-ssl https://raw.githubusercontent.com/rancher/rancher/${SSL_SCRIPT_COMMIT}/server/bin/update-rancher-ssl && \
+    chmod +x /usr/bin/update-rancher-ssl
+
+COPY lb-controller.sh /usr/bin/
+
 # So we do not write to the COW filesystem
 VOLUME /var/log/haproxy
 
 ENTRYPOINT ["/tini", "--"]
```

Comment:
this one is tricky. As you noticed, the entry point is "tini" that launches lb-controller process a special way (more details [here](https://github.com/krallin/tini). To test that thing work as expected, you'd have to:

1) Create LB service in Rancher using this image. Monitor the container logs and see if anything works fine.
2) One extra test to do - modify the LB service by either adding one more target service to it, or removing the existing one. Login inside the container and see how many haproxy processes you see by executing `ps aux | grep haproxy`(should be just one). If you see more than one, let me know